config/dockerfiles/linchpin/Dockerfile

```diff
@@ -1,9 +1,16 @@
 FROM fedora:28
 
 LABEL maintainer="rnester@redhat.com, cmays@redhat.com, ari@redhat.com"
```

Comment:
LABEL name="contrainfra/linchpin-executor" \
      maintainer="rnester@redhat.com, cmays@redhat.com, ari@redhat.com" \
      version="0.0.4" \
      description="A container to set up infrastructure instances"